src/video-interface.js

```diff
@@ -105,10 +105,40 @@ export class VideoInterface {
    *
    * @param {string} unusedMethod
    * @param {function(!./service/action-impl.ActionInvocation)} unusedHandler
-   * @param {ActionTrust} minTrust
+   * @param {ActionTrust} unusedMinTrust
    * @public
    */
-  registerAction(unusedMethod, unusedHandler, minTrust) {}
+  registerAction(unusedMethod, unusedHandler, unusedMinTrust) {}
+}
+
+
+/**
+ * @interface
+ */
+export class VideoInterfaceWithAnalytics extends VideoInterface {
```

Comment:
So this is now down to just 4 methods, do we still wanna keep it as a separate class? @aghassemi ?